.travis.yml

```diff
@@ -11,7 +11,7 @@ addons:
 language: go
 branches:
   only:
-    - master
+    - csi-v0.3
```

Comment:
Wouldn't this change prevent the linter and builds in the jobs list below not run against master? IOW, we want these jobs to run, but we do not want to deploy/push the master branch to quay, right? In that case, the change to deploy.sh should be enough, right?

Reply:
It does seem that we only need the changes to deploy.sh; we do want to keep running linter etc against master.

Reply:
yeah, my bad, we do need to run static checks on the master.